app/controllers/concerns/basic_authenticator.rb

```diff
@@ -19,7 +19,7 @@ def perform_basic_authn
       raise ApplicationController::Unauthorized, "Invalid username or password"
     rescue Errors::Authentication::InvalidOrigin
       raise ApplicationController::Forbidden, "User is not authorized to login from the current origin"
-    rescue Errors::Authentication::Security::UserNotAuthorizedInConjur
```

Comment:
Please follow suite here by setting  "Errors::Authentication::Security" outside so that it is "Err = Errors::Authentication::Security"

Reply:
*below*

Reply:
This class doesn't use this method. We do this only when we have a lot of the same error module in the class so we don't want to repeat ourselves.